src/components/Sidebar.tsx

```diff
@@ -24,7 +26,7 @@ interface SidebarProps {
   };
 }
 
-const navigationItems = [
+const baseNavigationItems = [
```

Comment:
const baseNavigationItems = [
  { icon: CheckSquare, label: "Tasks", href: "/tasks" },
  { icon: Network, label: "Stakgraph", href: "/stakgraph" },
  { icon: BarChart3, label: "Insights", href: "/insights" },
  { icon: Settings, label: "Settings", href: "/settings" },
];

Reply:
keep one source of truth for navigation